templates/create-daml-app/ui/.gitignore

```diff
@@ -0,0 +1 @@
+daml.js/
```

Comment:
I would suggest we drop this file and change the `daml.js/` entry in `templates/create-daml-app/.gitignore` into `ui/daml.js/`. That way, we need to ship less files and the project appears a tiny bit less complex.

Reply:
changed it, thanks!